src/auth/router.js

```diff
@@ -1,25 +1,25 @@
-"use strict";
+'use strict';
 
-const express = require("express");
-const passport = require("passport");
-const jwt = require("jsonwebtoken");
+const express = require('express');
+const passport = require('passport');
+const jwt = require('jsonwebtoken');
 
-const config = require("../config");
+const config = require('../config');
 const router = express.Router();
 
 const createAuthToken = user => {
   return jwt.sign({ user }, config.JWT_SECRET, {
     subject: user.name,
     expiresIn: config.JWT_EXPIRY,
-    algorithm: "HS256"
+    algorithm: 'HS256',
   });
 };
 
-const localAuth = passport.authenticate("local", { session: false });
+const localAuth = passport.authenticate('local', { session: false });
 
-router.post("/", localAuth, (req, res) => {
+router.post('/', localAuth, (req, res) => {
   const user = req.user.serialize();
-  const authToken = createAuthToken(req.user);
+  const authToken = createAuthToken({ _id: user.id, name: user.name });
```

Comment:
Does this remove password from authToken?

Reply:
@a-w-m Me storing the password in there was a bad idea.  I had to remove it because I don't use it for anything besides just adding it to there, the only I used in there was the user id.